zcash_primitives/src/transaction/components/sapling.rs

```diff
@@ -142,22 +181,54 @@ impl<A: Authorization> std::fmt::Debug for SpendDescription<A> {
     }
 }
 
+impl<A: Authorization> SpendDescription<A> {
+    /// Returns the commitment to the value consumed by this spend.
+    pub fn cv(&self) -> &ValueCommitment {
+        &self.cv
+    }
+
+    /// Returns the root of the Sapling commitment tree that this spend commits to.
+    pub fn anchor(&self) -> &bls12_381::Scalar {
+        &self.anchor
+    }
+
+    /// Returns the nullifier of the note being spent.
+    pub fn nullifier(&self) -> &Nullifier {
+        &self.nullifier
+    }
+
+    /// Returns the randomized verification key for the note being spent.
+    pub fn rk(&self) -> &PublicKey {
+        &self.rk
+    }
+
+    /// Returns the proof for this spend.
+    pub fn zkproof(&self) -> &A::Proof {
+        &self.zkproof
+    }
+
+    /// Returns the authorization signature for this spend.
+    pub fn spend_auth_sig(&self) -> &A::AuthSig {
+        &self.spend_auth_sig
+    }
+}
+
 /// Consensus rules (§4.4) & (§4.5):
 /// - Canonical encoding is enforced here.
 /// - "Not small order" is enforced in SaplingVerificationContext::(check_spend()/check_output())
 ///   (located in zcash_proofs::sapling::verifier).
-pub fn read_point<R: Read>(mut reader: R, field: &str) -> io::Result<jubjub::ExtendedPoint> {
+fn read_value_commitment<R: Read>(mut reader: R, field: &str) -> io::Result<ValueCommitment> {
```

Comment:
The `field` argument is redundant, since it is always `cv` (as implied now by the function name).